CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Swift 5.7
  
   Add a `--disable-testable-imports` flag to `swift test` with which tests are built without the testability feature (`import @tstable` disabled).
 
+* [#4135]
```

Comment:
sorry also need to add a link, e.g. https://github.com/apple/swift-package-manager/pull/4135/commits/6e3ed66afad54dcd2cee8390b58ace2164117912#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL202